app/components/dashboard/deposit_progress_component.rb

```diff
@@ -6,30 +6,31 @@ module Dashboard
   class DepositProgressComponent < ApplicationComponent
     sig { params(work: Work).void }
     def initialize(work:)
-      @work = work
+      @form = T.let(WorkForm.new(work), WorkForm)
+      @form.valid?
```

Comment:
I was a bit surprised to see `#initialize` end with a question-mark method. What does this do? Populate the errors array? If needed, can you drop a comment to that effect?